ztag/stream.py

```diff
@@ -176,17 +197,31 @@ def __init__(self, logger=None, destination=None, *args, **kwargs):
         else:
             raise Exception("invalid destination: %s" % destination)
         host = os.environ.get('KAFKA_BOOTSTRAP_HOST', 'localhost:9092')
+        processes = os.environ.get('ZTAG_KAFKA_PROCESSES', 2)
         self.main_producer = KafkaProducer(bootstrap_servers=host)
         self.cert_producer = KafkaProducer(bootstrap_servers=host)
+        self.queue = multiprocessing.Queue()
+        self.processes = list()
+
+        for pnum in xrange(0, int(processes)):
```

Comment:
This looks like you're sharing a kafka producer across processes? This seems dangerous unless the library explicitly guarantees that this is safe. I would have expected that each process starts its own producer.